app/views/account_request_mailer/confirmation.html.erb

```diff
@@ -42,8 +42,17 @@
 </p>
 
 <p>
-    Finally, we made a getting started video with detailed directions on setting up your essentials bank in Human Essentials, check it out
-    <a href='https://www.youtube.com/watch?v=fwo3WKMGM_4&feature=youtu.be'>here</a>! Please let us know when you would like to begin using Human Essentials and we will set you up and send you a welcome email.
+    Finally, we've made a series of getting started videos with detailed directions on setting up your essentials bank in Human Essentials, check them out:
+<br>
+  <a href='https://youtu.be/w53iaJtlQUo'>Part 1</a><br/>
+  <a href='https://youtu.be/3-CF0wsG0h4'>Part 2</a><br/>
+  <a href='https://youtu.be/WkJahWsdZzc'>Part 3</a><br/>
+  <a href='https://youtu.be/STl0MfLQYbc'>Part 4</a><br/>
+  <a href='https://youtu.be/oREioR1Qg6c'>Part 5</a><br/>
+  <a href='https://youtu.be/feg59oW6V3M'>Part 6</a><br/>
+  <a href='https://youtu.be/wyD034nvqag'>Part 7</a><br/>
+  <a href='https://youtu.be/v3AyrL0pHw0'>Part 8</a><br/>
```

Comment:
Is there a way to turn this into a playlist on youtube so we can make this one link that starts at Part 1?

Reply:
In each of the videos in the description there are links to all the others.

Reply:
Would it make sense to also send the list also to everyone who has signed up in, say, the last 3 months?

Reply:
Having this list in the Read.me might be a good idea, too.

Reply:
Do we want to add that these are current as of May 2022 -- there may be new features that aren't covered?